src/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Contract, constants } from 'ethers'
+
+import { Provider } from '@ethersproject/providers'
+
+/**
+ * Returns the broker address from the Celo registry
+ * @param provider an ethers provider
+ * @returns the broker address
+ */
+export async function getBrokerAddressFromRegistry(
+  provider: Provider
+): Promise<string> {
+  const celoRegistryAddress = '0x000000000000000000000000000000000000ce10'
+  const brokerIdentifier = 'Broker'
+
+  const registryAbi = [
+    'function getAddressForString(string calldata identifier) external view returns (address)',
+  ]
```

Comment:
consider moving these to a `consts` file

Reply:
I generally use `consts` files for constants that are either reused or when I have several that can be grouped. Otherwise I prefer keeping them in the place they are used. Happy to change it others have a similar opinion here though